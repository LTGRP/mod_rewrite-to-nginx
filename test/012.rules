RewriteRule ^localpath(.*) /otherpath$1 [R]
