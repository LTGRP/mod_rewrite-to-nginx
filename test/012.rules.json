[
    {
        "type": "rewrite",
        "match": "^localpath(.*)",
        "target": "/otherpath$1",
        "redirect": true,
        "code": 302
    }
]
