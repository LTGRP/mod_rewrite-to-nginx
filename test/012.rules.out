rewrite ^localpath(.*) /otherpath$1 redirect;
